doc/Language/exceptions.pod6

```diff
@@ -84,9 +93,15 @@ methods apply to C<$_>, which holds the C<Exception> within the C<CATCH> block.
          }
     }
 
-Note that the match target is a role. To allow user defined exceptions to match
-in the same manner, they must implement the given role. Just existing in the
-same namespace will look alike but won't match in a C<CATCH> block.
+Note that the match target is a role. To allow user defined exceptions to
+match in the same manner, they must implement the given role. Just existing
+in the same namespace will look alike but won't match in a C<CATCH> block.
+
+Note that the C<CATCH> block semantics apply to the B<entire> lexical scope
```

Comment:
Starts with the same word as above. Consider alternatives "Please also" or "Note also"